content/documentation/rpc/chain/get_transaction_stream.mdx

```diff
@@ -22,34 +22,38 @@ Streams transactions from a head sequence given an incoming view key
   head: {
     sequence: number
   }
-  block: {
-    hash: string
-    previousBlockHash: string
-    sequence: number
-    timestamp: number
-  }
-  transactions: Array<{
-    hash: string
-    isMinersFee: boolean
-    notes: Array<{
-      assetId: string
-      assetName: string
-      hash: string
-      value: string
-      memo: string
-    }>
-    mints: Array<{
-      assetId: string
-      assetName: string
-      value: string
-    }>
-    burns: Array<{
-      assetId: string
-      assetName: string
-      value: string
-    }>
-  }>
+  block: RpcBlockHeader
+  transactions: Transaction[]
 }
 ```
 
+Sub Objects: 
+
+```js
+interface Note {
+  assetId: string
+  /**
+   * @deprecated Please use getAsset endpoint to get this information
+   */
+  assetName: string
+  hash: string
+  value: string
+  memo: string
+}
+
+interface Transaction {
+  hash: string
+  isMinersFee: boolean
+  notes: Note[]
+  mints: RpcMint[]
+  burns: RpcBurn[]
+}
+```
```

Comment:
not sure if this the best way to do it. Maybe I think I didn't find a way to combine this with RpcTransaction